src/FingerprintPro.ServerSdk/Sealed.cs

```diff
@@ -0,0 +1,154 @@
+using System.IO.Compression;
```

Comment:
Is it correct in terms of .NET best practice to keep this file in the root?

Reply:
The directories must match the namespace, but I think it's okay to keep this class in root.